lib/sentry/req_client.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Sentry.ReqClient do
+  @behaviour Sentry.HTTPClient
+
+  @moduledoc """
+  The built-in HTTP client.
+
+  This client implements the `Sentry.HTTPClient` behaviour.
+
+  It's based on the [Req](https://github.com/wojtekmach/req) HTTP client,
+  which is an *optional dependency* of this library. If you wish to use another
+  HTTP client, you'll have to implement your own `Sentry.HTTPClient`. See the
+  documentation for `Sentry.HTTPClient` for more information.
+  """
+
+  @impl true
+  def post(url, headers, body) do
+    opts =
+      Sentry.Config.req_opts()
+      |> Keyword.put(:decode_body, false)
+      |> Keyword.put(:url, url)
+      |> Keyword.put(:body, body)
```

Comment:
Req decodes responses automatically. I choose to disable body decoding so the users can make use of the `json_library` option.